### PR TITLE
Add shellcheck (docker) pre-commit hook

### PR DESCRIPTION
**Pull request by jooola**
This PR needs #2668 to be merged first.

Next iteration on the pre-commit setup: This adds a local shellcheck pre-commit hook using the https://hub.docker.com/r/koalaman/shellcheck/ docker image.

### CODE REVIEW

File: .github/scripts/push-docker.sh [hunk 1]
```diff
@@ -2,28 +2,48 @@
...
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_arm64" \
+  "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_armhf"
+
+docker manifest push "ghcr.io/savonet/liquidsoap:$TAG"
+
+docker manifest create \
+  "ghcr.io/savonet/liquidsoap-alpine:$TAG" \
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_alpine_amd64" \
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_alpine_arm64" \
+  --amend "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_alpine_armhf"
```
Comment (jooola) [inline .github/scripts/push-docker.sh:47]: Unsure if there is some kind of inconsistency. The last manifest create command has 3 --amend while others don't.

Comment (toots) [reply .github/scripts/push-docker.sh:47]: Looks like it works both ways but let's make it consistent.

Comment (toots) [inline .github/scripts/push-docker.sh:47]:
```suggestion
  "ghcr.io/savonet/liquidsoap-ci-build:${TAG}_alpine_armhf"
```

File: tests/media/mk_stream_test.sh [hunk 2]
```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 
-BASEPATH=$0
-BASEDIR=$(dirname $0)
-CWD=$(cd $BASEDIR && pwd)
+# BASEPATH="$0"
```
Comment (toots) [inline tests/media/mk_stream_test.sh:3]:
```suggestion
```

**Review by toots — APPROVED** [review]

**Pull request merged**